src/lib/openpath/normalizeData.js

```diff
@@ -0,0 +1,55 @@
+import { GEOJSON_DOCTYPE } from 'src/doctypes'
+import { initPolyglot } from 'src/lib/services'
+import { computeAggregatedTimeseries } from 'src/lib/timeseries'
+import { getSectionsFromTrip } from 'src/lib/trips'
+import { buildSettingsQuery } from 'src/queries/queries'
+
+const { t } = initPolyglot()
+
+/**
+ * @typedef {import("cozy-client/dist/index").CozyClient} CozyClient
+ * @typedef {import('./types').TimeseriesGeoJSON} TimeseriesGeoJSON
+ * @typedef {import('./types').RawGeoJSON} RawGeoJSON
+ */
+
+/**
+ * Normalize trips to timeseries
+ *
+ * @param {Cozyclient} client - The cozy client instance
+ * @param {RawGeoJSON>} trips - The trip to normalize
+ * @returns {Array<TimeseriesGeoJSON>} the normalized timeseries
+ */
+export const normalizeTrips = async (client, trips, { device }) => {
```

Comment:
Could you add the third argument to the JSDoc ? 😇

Reply:
Good catch, I added the 3rd after making the jsdoc